client/src/modules/assets/templates/action.cell.html

```diff
@@ -0,0 +1,77 @@
+<div class="ui-grid-cell-contents text-right" uib-dropdown dropdown-append-to-body uib-dropdown-toggle ng-if="!row.groupHeader">
+  <a href>
+    <span data-method="action" translate>FORM.BUTTONS.ACTIONS</span>
+    <span class="caret"></span>
+  </a>
+
+  <ul data-action="{{ rowRenderIndex}}"  class="dropdown-menu-right" bh-dropdown-menu-auto-dropup uib-dropdown-menu>
```

Comment:
You probably copied-pasted this from somewhere.  We don't use the `{{ rowRenderIndex }}` anymore.  Please remove it.

Reply:
Done